views/default/forms/admin/plugins/sort.php

```diff
@@ -8,7 +8,9 @@
  */
 
 echo '<div class="mtm">';
+echo '<label for="admin-plugins-sort" class="hidden">' . elgg_echo('sort') . '</label>';
```

Comment:
It is pretty easily shimmed with "[hidden] { display: none !important }" somewhere in the code, no?